packages/google_maps_routes_api/lib/src/types/navigation_instruction.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
One empty line after copyright (not sure if these are needed, but better be sure)